From 5c4e96c9d13576b3f6324130fa93c45c703b0d3c Mon Sep 17 00:00:00 2001
From: Maxime Ripard <maxime.ripard@free-electrons.com>
Date: Wed, 24 Jul 2013 09:12:08 +0200
Subject: [PATCH 12/12] drivers: net: sun4i-emac: select MDIO_SUN4I

The EMAC driver can't work without its associated PHY driver. Reflect
this in the Kconfig options by selecting it.

Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 111cc5da2a95568ca20a5f86f43ae328c069e455)
---
 drivers/net/ethernet/allwinner/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/allwinner/Kconfig
+++ b/drivers/net/ethernet/allwinner/Kconfig
@@ -28,6 +28,7 @@ config SUN4I_EMAC
 	select NET_CORE
 	select MII
 	select PHYLIB
+	select MDIO_SUN4I
         ---help---
           Support for Allwinner A10 EMAC ethernet driver.
 
